docs/design.md

```diff
@@ -1091,6 +1136,8 @@ The gossip protocol itself contains two primary components:
 
 # Node Accounting
 
+[Replace with description of MVCC, store-level aggregation, recorder,
```

Comment:
@mrtracy would be great if you could replace `Node Accounting` section with an `Accounting`...